azure-kusto-data/azure/kusto/data/client_base.py

```diff
@@ -4,20 +4,23 @@
 import uuid
 from copy import copy
 from datetime import timedelta
-from typing import Union, Optional, Any, NoReturn, ClassVar
+from typing import Union, Optional, Any, NoReturn, ClassVar, TYPE_CHECKING
```

Comment:
nit: did the linter go nuts on your PR?
If so, can we push a PR with forced linting for all files in the repository?

Reply:
There was a new version of black, so that's probably what has changed